webrender/src/device/gfx/device.rs

```diff
@@ -1641,41 +1515,9 @@ impl<B: hal::Backend> Device<B> {
     }
 
     fn draw(&mut self) {
+        assert!(self.inside_renderpass);
```

Comment:
I'll look into upstreaming the render pass concept in WR, once we make it work here.